plugins/aglight/plugin.json

```
@@ -0,0 +1,30 @@
+{
+  "$schema": "../../schemas/plugin.json",
+
+  "name": "AGLight v1.0.0",
```

One more thing: A version string in the plugin name refers to the supported file format version, not the plugin version. I guess you can just drop it in this case. See e.g. https://open-fixture-library.org/about/plugins/qlcplus_4.12.2 how the QLC+ format version, the import plugin version and the export plugin version differ.

```suggestion
  "name": "AGLight",
```

Note that you also need to run `make` to keep `plugins.json` updated.